Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -29,6 +29,35 @@
 
 namespace AZ::Internal
 {
+    //! References the settings key to set the project path via *.setreg(patch) file
+    //! Lowest Priority: Will be overridden by the Project Scan Up Key value
+    //! This setting shouldn't be used be at all
+    //! Reason: The path to the project is needed to determine where the project is located,
+    //! which engine the project is using as well as which gems the project is using.
+    //! Therefore this setting would need to be set within a registry file(.setreg)
+    //! that is available without knowing the project root at all.
+    //! The only location registry files can be relying found while the project path is not
```

Comment:
```suggestion
    //! The only location registry files can be reliably found while the project path is not
```

Reply:
> Just verifying, this doesn't have an effect on how pak builds work, right?

Correct, Pak builds will see still fallback to using the executable directory as the project root(https://github.com/o3de/o3de/blob/2d975234922a105a4ba604ca92cf48482e3fadeb/Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp#L816-L835), when a `project.json` file is not found by scanning upwards.